src/app/src/main/res/layout/book_preview.xml

```diff
@@ -23,7 +23,7 @@
         android:layout_marginStart="8dp"
         android:layout_marginTop="8dp"
         android:layout_marginEnd="16dp"
-        android:text="Book Title"
+        android:text="@string/book_title"
```

Comment:
Nice job making these shared string resources. A lot of pages will share our strings, so we want to continue doing this.